remote_attestation/java/src/com/google/oak/remote_attestation/ClientHandshaker.java

```diff
@@ -91,88 +75,104 @@ public ClientHandshaker(byte[] expectedTeeMeasurement) {
   /**
    * Initializes the remote attestation handshake by creating an {@code ClientHello} message.
    *
-   * Transitions {@code ClientHandshaker} state from {@code Initializing} to
-   * {@code Attesting} state.
+   * Transitions {@code ClientHandshaker} state from {@code State.Initializing} to
+   * {@code State.ExpectingServerIdentity} state.
    */
   public byte[] createClientHello() throws IllegalStateException, IOException {
```

Comment:
Pre-existing: IllegalStateException is a RuntimeException. Does it have to be specified in the signature? 
Also below in `processServerIdentity`.

Reply:
Updated to `throws RuntimeException`

Reply:
It is an unchecked exception. You don't have to specify it in the signature. I think `public byte[] createClientHello() throws IOException {` should be enough.